torch/jit/__init__.py

```diff
@@ -708,5 +710,32 @@ def __dir__(self):
         keys = [key for key in keys if not key.isdigit()]
         return keys
 
+
+class _ConstSequential(ScriptModule):
```

Comment:
What is the reason why this can't just subclass `ConstModuleList` and implement only forward?